src/components/KTabs/KTabs.vue

```diff
@@ -89,6 +95,8 @@ const emit = defineEmits<{
 
 const activeTab = ref<string>(props.modelValue ? props.modelValue : props.tabs[0]?.hash)
 
+const isNav = computed((): boolean => props.tabs.every(tab => tab.to))
```

Comment:
question: what happens if some tabs have a `to` prop and others don't?

Reply:
Then `role="tablist"` will be used. In other words, means it's a tabs element with some tabs being the links (vs if all tabs are links then it's a navigation element `role="navigation"`).